beta_dist_plot.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    aa = [0.1, 0.1, 1.0, 2.0, 2.0]
</pre><p>can we use a_list instead of aa? PEP8</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done